### DISCUSSION TITLE
first-contributions -  issue with compare and pull request at end of the steps

### DISCUSSION THREAD

**Discussion by ghost** [community/community#143277] [New to GitHub]
Hi,
I did everything according to the steps. All that was left was to press "Compare and Pull Request". But that button _did not show up_.

I had a few hiccups but everything was ok. Even the errors on the list of "If there are errors" didn't show up. 

I want to fix the situation but I can't identify the issue.
[screencapture-github-zshubaily-first-contributions-tree-main-2024-11-01-12_55_13.pdf](https://github.com/user-attachments/files/17598469/screencapture-github-zshubaily-first-contributions-tree-main-2024-11-01-12_55_13.pdf)

[zshubaily](https://github.com/zshubaily)
/
[first-contributions](https://github.com/zshubaily/first-contributions)

Does anyone have advice?

Thanks

**Comment by CemWebDev** [comment]
Hi @zshubaily,

If the "Compare and Pull Request" button is not showing up, here are a few steps to check:

1. **Ensure you're on the correct branch**: Make sure you've created and made changes on a new branch in your forked repository, rather than directly on the `main` branch.

2. **Confirm that you've pushed your changes to GitHub**: If you made changes locally, be sure to push them to GitHub using:
   ```bash
   git push origin <branch-name>

**Reply by ghost** [reply]
Thanks @CemWebDev 

But it looks like this is a lost cause. I've tried many things and methods but I'm in a tangle. I did merging and other experiments. It seems I'll have to find another novice project to train more.

**Comment by ghostinhershell** [comment]
Hi @zshubaily, thanks so much for reaching out in our community 👋🏾 

Are you still running into this issue on your end? Sometimes [clearing your cache and cookies](https://kb.iu.edu/d/ahic) does the trick. This also could have been caused by a service outage. You can check status.github.com to confirm in the future if a page load error you're seeing was caused by an outage.

**Comment by ghost** [comment]
Hi @ghostinhershell ,

Thank you for your follow-up. I've tried implementing some changes as can be seen from the attached. But still can't see the button Compare and Pull Request.
![Screenshot 2024-11-04 215248](https://github.com/user-attachments/assets/6dcbc480-8171-4068-aacf-06fea380e085)

**Comment by Roshanjossey** [comment]
Hi @zshubaily, maintainer of first contributions here.

I see that your main branch is 2 commits ahead of parent repo

![image](https://github.com/user-attachments/assets/e6f4610f-3532-45b9-9888-117b8b483348)


If you click on that link, you'll be taken to compare page

https://github.com/firstcontributions/first-contributions/compare/main...zshubaily:first-contributions:main

You can see create pull request button here 
![image](https://github.com/user-attachments/assets/8d0eda7b-47ce-4a8b-8757-3e05fda83931)
